runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/util/BoundedQueueExecutor.java

```diff
@@ -104,6 +117,21 @@ public void forceExecute(Runnable work, long workBytes) {
     executeLockHeld(work, workBytes);
```

Comment:
can you rename executeLockHeld to executeMonitorHeld to clarify since we have both monitor and synchronized lock now?